### PR TITLE
Static configuration options for EntityView.

### PR DESCRIPTION
**Pull request by dcervelli**
Extract constants as editable static variables so users can tune `EntityView` behavior. In particular, this allows users to:

1. set a larger velocity approximate time delta when working with Entities that lack enough precision with 0.001s time steps,
2. change the multiplier for determining if an object is in "deep space" (this is likely what was being asked for in #8426).

I think a better long-term approach may be to allow entities to provide velocity information and/or specify the kind of camera reference frame they prefer. However, this seems like a reasonable, minimal (backwards compatible) change.

This provides a workaround for #9484.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thank you so much for the pull request @dcervelli! I noticed this is your first pull request and I wanted to say welcome to the Cesium community!

The [Pull Request Guidelines](https://github.com/CesiumGS/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines) is a handy reference for making sure your PR gets accepted quickly, so make sure to skim that.

* :x: Missing `CONTRIBUTORS.md` entry.
    * Please add yourself to the [contributors](https://github.com/dcervelli/cesium/blob/dc/entity-view-config/CONTRIBUTORS.md) file!
* :x: Missing CLA.
    * Please send in a [Contributor License Agreement](https://github.com/CesiumGS/cesium/blob/master/CONTRIBUTING.md#contributor-license-agreement-cla) (CLA) and comment back here to let us know to check this!
    * Maintainers, [here's how to check](https://github.com/CesiumGS/cesium/blob/master/Documentation/Contributors/CLAs/README.md) for a submitted CLA.
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/dc/entity-view-config/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/master/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

File: Source/DataSources/EntityView.js [hunk 1]
```diff
@@ -314,7 +334,6 @@ function EntityView(entity, scene, ellipsoid) {
   this._mode = undefined;
 
   this._lastCartesian = new Cartesian3();
-  this._defaultOffset3D = undefined;
```
Comment (dcervelli) [inline Source/DataSources/EntityView.js:317]: This didn't do anything.

Comment (ebogo1) [reply Source/DataSources/EntityView.js:317]: We usually declare class properties in the constructor so they don't have to be dynamically allocated later by a setter.

Comment (dcervelli) [reply Source/DataSources/EntityView.js:317]: Sorry, I'm confused. I thought `EntityView._defaultOffset3D` was a _static_ value initialized via the setter by `EntityView.defaultOffset3D = ...` and the `this.defaultOffset3D` referred to here is never read/unused.

In any case, this line doesn't really have anything to do with the PR and I'm happy to put it back in if you want to accept this change.

Comment (ebogo1) [reply Source/DataSources/EntityView.js:317]: Yeah, I think the idea was to have the property already part of `EntityView` by the time the setter does `this._defaultOffset3D = ...`. @mramato @lilleyse Is this correct?

Comment (lilleyse) [reply Source/DataSources/EntityView.js:317]: I think @dcervelli's change is correct. The setter is setting a static variable whereas `this._defaultOffset3D = undefined` is an unused `EntityView` instance property.

It looks look it was added in https://github.com/CesiumGS/cesium/commit/d4accd7b993af7a357e4d0eaa76f6f45fd86e33e but I'm not sure why. Do tests still pass?

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @dcervelli! 

No one has commented on this pull request in 90 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 90 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @dcervelli! 

No one has commented on this pull request in 90 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 90 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @dcervelli! 

No one has commented on this pull request in 90 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 90 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @dcervelli! 

No one has commented on this pull request in 90 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 90 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @dcervelli! 

No one has commented on this pull request in 90 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 90 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @dcervelli! 

No one has commented on this pull request in 90 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 90 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @dcervelli! 

No one has commented on this pull request in 90 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 90 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @dcervelli! 

No one has commented on this pull request in 90 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 90 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by ggetz** [comment]
Hi @dcervelli, I'm closing this issue due to inactivity. 

Please feel free to open a new PR, and we'll give this a fresh review. Thanks!

**Pull request closed without merging**